### PR TITLE
Support SignalR trigger return value

### LINKED ISSUE #1496: The SignalR triggered function return value not working
The SignalR `Microsoft.Azure.WebJobs.Host.Triggers.ITriggerData.ReturnValueProvider.SetValueAsync(object value, CancellationToken cancellationToken)`  is always invoked with parameter `value=null` for isolated worker. This means the return value of SignalR triggered function without output binding annotated is not working at all. However, it works in the in-process model and the isolated process model of `HTTP` trigger. 

```cs
[Function("Broadcast")]
public async Task<NewMessage> Broadcast(
[SignalRTrigger(nameof(Functions), "messages", "broadcast", "message")] SignalRInvocationContext invocationContext, string message)
{
    return new NewMessage(invocationContext, "this is returned from function");
}
```

The [Worker.Extensions.Http](https://github.com/Azure/azure-functions-dotnet-worker/tree/main/extensions/Worker.Extensions.Http)` trigger is quite simple that has nothing to do with the return value handling. Need more investigation on what makes the difference. Any insight would be appreciated.

### LINKED ISSUE #1497: Question: When are security patches applied to azure functions?
When are security patches applied to azure functions? Is there a timeline?
When will functions get patched from dotnet 7.0.4 to 7.0.5?

Thank you!

### PR DESCRIPTION
**Pull request by Y-Sindo**
Fix the return value of SignalR trigger not working issue

resolves #1496
### Pull request checklist

* [x] My changes **do not** require documentation changes
  * [ ] Otherwise: Documentation issue linked to PR
* [ ] My changes **should not** be added to the release notes for the next release
  * [x] Otherwise: I've added my notes to `release_notes.md`
* [x] My changes **do not** need to be backported to a previous version
  * [ ] Otherwise: Backport tracked by issue/PR #issue_or_pr
* [x] I have added all required tests (Unit tests, E2E tests)

### CODE REVIEW

File: extensions/Worker.Extensions.SignalRService/release_notes.md [hunk 1]
```diff
@@ -1,8 +1,6 @@
 ## What's Changed
 
-<!-- Please add your release notes in the following format:
-- My change description (#PR/#issue)
--->
+- Fix SignalR trigger return value not working issue.
```
Comment (satvu) [inline extensions/Worker.Extensions.SignalRService/release_notes.md:3]: Could you move this below the header (line 5) and update the header to say 1.15.0? Thanks!

**Review by satvu — APPROVED** [review]

**Comment by satvu** [comment]
@Y-Sindo Can you merge main to your branch to make sure you have [this change](https://github.com/Azure/azure-functions-dotnet-worker/commit/97e7048bf26b7da6d159a2c9143f42f8f91a7ef9) included which will allow the pipeline to run? Thanks!

File: extensions/Worker.Extensions.SignalRService/release_notes.md (hunk 1 above)
Comment (fabiocav) [reply extensions/Worker.Extensions.SignalRService/release_notes.md:3]: Labeling for author feedback based on this request.

**Comment by waqarzafar** [comment]
The fixes have been applied to .NET6 package: Microsoft.Azure.WebJobs.Extensions.SignalRService whereas the same issue exists in .NET8 version of this package: Microsoft.Azure.Functions.Worker.Extensions.SignalRService. Please kindly confirm when will the fix be propagated to this library for .NET8 applications

**Comment by Y-Sindo** [comment]
@waqarzafar The fix is already released. Please update your SignalR worker extensions to 1.15.0. Thanks!

**Pull request merged**